client/state/ui/editor-deprecation-dialog/actions.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import { EDITOR_DEPRECATION_DIALOG_IS_SHOWING } from 'state/action-types';
+
+export function showEditorDeprecationDialog() {
```

Comment:
It doesn't look like this is ever called...

Reply:
Good point, removed.